### ISSUE TITLE
Suggestion: Dynamic Page Titles

### ISSUE THREAD

**Issue by batannius** [EchoLeague/Echo-League-Website-Testing#43]
TLDR; please add dynamic page titles for every page.

---
As a team organizer it is very likely that you'll have many EchoLeague.gg tabs open at once, maybe to just look at different stuff and maybe to prepare your own team for the next game, so you have 7 player- and 1 team tab open, for example.

So it would be nice and save a lot of clicking time to identify which tab is which page, which players and which teams you have open and what other sites additional to that.
And since it's very easy to identify the EchoLeague.gg tabs via thumbnail, it wouldn't be necessary to show the actual website name in any tab except for HOME.

---
Could be just simple title formats like this:

![echotabs](https://cloud.githubusercontent.com/assets/26058743/23357654/164484e8-fcdf-11e6-94b7-5534a473e64f.png)
